source/jormungandr/jormungandr/stat_manager.py

```diff
@@ -160,6 +161,13 @@ def _manage_stat(self, start_time, call_result):
         self.fill_result(stat_request, call_result)
         self.publish_request(stat_request)
 
+    def fill_info_response(self, stat_info_response, call_result):
+        """
+        store data from response of all requests
+        """
+        if 'pagination' in call_result[0]:
```

Comment:
you may want to check also 'items_on_page', no?
`if 'pagination' in call_result[0] and 'items_on_page' in call_result[0]['pagination']`